src/components/strategies/create/BuySellBlock/index.tsx

```diff
@@ -134,7 +140,15 @@ export const BuySellBlock: FC<Props> = ({
       )}
 
       <LimitRangeSection
-        {...{ base, quote, order, buy, title, inputTitle, isOrdersOverlap }}
+        {...{
```

Comment:
what changed here? Did the formatter rules change? This looks like more noise in the review

Reply:
When added new props it automatically change the format, but when removing it is was still that way.
anyway, fixed that now.